Rakefile

```diff
@@ -1,6 +1,8 @@
+require "json"
+
 def e(cmd)
   puts cmd
-  system(cmd) || abort("Error running `#{cmd}`")
+  system(cmd) || raise("Error running `#{cmd}`")
```

Comment:
link me to https://github.com/square/p2/pull/733#discussion_r97616796 so that I can remind myself of why this changed